lib/eex/lib/eex/tokenizer.ex

```diff
@@ -27,13 +17,6 @@ defmodule EEx.Tokenizer do
 
   Or `{:error, message, %{column: column, line: line}}` in case of errors.
   """
-  @spec tokenize(
```

Comment:
I guess we don't need it here anymore since we are documenting the public version of it, right?